tests/gmbitbang/test_packer.c

```diff
@@ -1,21 +1,26 @@
 #include <stdio.h>
 #include <stdint.h>
 
-typedef struct {
-  uint32_t RIR;  /*!< CAN receive FIFO mailbox identifier register */
-  uint32_t RDTR; /*!< CAN receive FIFO mailbox data length control and time stamp register */
-  uint32_t RDLR; /*!< CAN receive FIFO mailbox data low register */
-  uint32_t RDHR; /*!< CAN receive FIFO mailbox data high register */
-} CAN_FIFOMailBox_TypeDef;
+#define CANPACKET_DATA_SIZE_MAX 8
+typedef struct __attribute__((packed)) {
```

Comment:
should be able to just include a header file with these definitions from the board code